src/lib/holocene/select/native-select.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+  export let label = '';
+  export let id: string;
+  export let placeholder: string;
+  export let value = '';
+  export let options: { label: string; value: string }[] = [];
+</script>
+
+{#if label}
+  <label for={id} class={$$props.labelClass}>{label}</label>
+{/if}
+<select {value} {id} class={$$props.class} {placeholder} on:select>
+  {#each options as option}
+    <option value={option.value}>{label}</option>
```

Comment:
I think I would prefer this to just be a default slot and compose the options myself. That way we could expose more bind values and generally have more control over the sub components.